app/Main.hs

```diff
@@ -42,5 +63,17 @@ fps :: Int
 fps = 60
 
 -- | Main
+playIO' :: Display
+       -> Color
+       -> Int
+       -> World
+       -> (World -> IO Picture)
+       -> (Event -> World -> IO World)
+       -> (Float -> World -> IO World)
+       -> IO ()
+playIO' = playIO
+
 main :: IO ()
-main = play window background fps initialState renderGame handleKeys update
+main = do
+  (pictureImage :: Picture) <- loadBMP "/media/akhesa/16e1988c-fe98-4a2e-b528-320abdc3d132/akhesa/projects/haskell-breakout/library/purpleBrick.bmp"
```

Comment:
I'd suggest using a relative path. Using `stack exec`, the working directory should be the root folder of the project, so just `./library/purpleBrick.bmp` should do.